swap_meet/vendor.py

```diff
@@ -1,2 +1,68 @@
+from swap_meet.item import Item
 class Vendor:
-    pass
\ No newline at end of file
+    
+    def __init__(self, inventory=None):
+        # self.inventory = inventory
+        if inventory is None:
+            inventory = []
+        self.inventory = inventory
+    
+    def add(self, item):
+        self.item = item
```

Comment:
👀 We don't need to store a reference to the `item` in the Vendor directly. Adding it to the inventory is sufficient.

And in general, we should avoid assigning to `self` attributes that aren't initialized in the constructor, as this effectively changes the attributes that are available depending on which methods have been called, which can be confusing. There are linting tools (tools to report code issues that aren't necessarily errors, but are somewhat suspect) that will warn about assigning to a new attribute like this.